test/Dapr.Actors.Test/Runtime/ActorRuntimeOptionsTests.cs

```diff
@@ -113,5 +112,30 @@ public void SettingJsonSerializerOptionsToNull_Fails()
 
             action.Should().Throw<ArgumentNullException>();
         }
+
+        [Fact]
+        public void SettingReentrancyConfigWithoutMaxStackDepth_Succeeds()
```

Comment:
I would prefer not having tests like this that just set properties and read them back. It's testing code that we *didn't* write.

Reply:
Fair enough. I included them as the other values had them. Would you like me to actually remove at least my new tests here?

Reply:
Yes, I think it's better just to not have things like this.